### PR TITLE
Grant proposal - Study & contributions on decentralized evaluation

### PR DESCRIPTION
**Pull request by PhilH**
The proposal aims at:
- Delivering a comprehensive study to explore and establish effective economic models for rewarding contributions in open source software, specifically tailored for the Gno.land ecosystem.
- Engaging with Gno.land extended team & actively contributing to the current research on governance and DAO/subDAO organizational design.

### CODE REVIEW

File: EvaluationDAO-Study.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- Phase 2: Exploring and collecting material from papers and interviews with subject matter experts (up to 5)
+- Phase 3: Compilation, synthesis, drafting of the study
+- Phase 4: Review and revision
+
+Regular updates will be arranged at any frequency Gno.land core team may see fit.
+The active participation of representative of Gno.land core team/DAOs is expected, especially in Phase 1 & 3.
+
+### Idea for Fair Funding
+
+We expect the total effort of the study to represent 20 and 35 man-days of work over 2 to 3 months, plus up to 5 days of complementary review with RnDAO.
```
Comment (moul) [inline EvaluationDAO-Study.md:78]:
```suggestion
We expect the total effort of the study to between represent 20 and 35 man-days of work over 2 to 3 months, plus up to 5 days of complementary review with RnDAO.
```

I guess?

Comment (PhilH) [reply EvaluationDAO-Study.md:78]: Both can work and mean the same, but if we add "between" for the sake of clarity, it should be after "represent" :)
I'll do it!

Comment (PhilH) [reply EvaluationDAO-Study.md]: Can you check again @moul ?

**Review by MichaelFrazzy — COMMENTED** [review]
I'd love to set up a call to go over everything here! I think it'd be extremely effective/efficient if we synced up any efforts, looking forward to your thoughts and talking again soon.

**Pull request merged**